v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-governance.md

  - blockchain
  - atom
---

> Governance is the ability for ATOM stakers to vote on proposals that affect the evolution of the network. Proposals can be submitted by any ATOM holder, but only proposals that come with a sufficient deposit (current minimum deposit is 512 ATOM) are eligible to be voted on. Deposits can be crowdfunded and need not be entirely provided by the user submitting the proposal. 

Reviewer: ```suggestion
Governance is the ability for ATOM stakers to vote on proposals that affect the evolution of the blockchain network. Any ATOM holder can submit proposals, but only proposals that come with a sufficient deposit (current minimum deposit is 512 ATOM) are eligible to be voted on. Deposits can be crowdfunded and do not need to be entirely provided by the user submitting the proposal.
```

> Once a proposal gets sufficient deposit, it enters the voting period, which lasts for 14 days. Most block explorers show the currently active proposals. ATOM stakers can vote on such proposals, with a voting power proportional to their amount of staked ATOM. If they do not vote, they inherit the vote from the validator(s) they delegated to. Voting can be performed via most [Wallets].

Reviewer: ```suggestion
After a proposal gets a sufficient deposit, it enters the 14-day voting period. Most block explorers show the currently active proposals. ATOM stakers can vote on active proposals, with a voting power that is proportional to their amount of staked ATOM. If the ATOM stakers do not vote, they inherit the vote from the validators they delegated to. Voting can be performed using most [Wallets].
```
